lib/utils.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+
+/**
+ * Deletes a file from the filesystem if it exists.
+ * Does nothing if it doesn't do anything.
+ * @param {string} pathToFile  path to file to delete
+ */
+function cleanFile(pathToFile) {
```

Comment:
This is fine for this PR --  but worth trying out `fs-extra` as it has this functionality built into the `.remove()` method.